orchagent/fdborch.cpp

```diff
@@ -1201,12 +1379,21 @@ bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name,
 
     FdbData storeFdbData = fdbData;
     storeFdbData.bridge_port_id = port.m_bridge_port_id;
+    // overwrite the type and origin
+    if ((fdbData.origin == FDB_ORIGIN_MCLAG_ADVERTIZED) && (fdbData.type == "dynamic_local"))
+    {
+        //If the MAC is dynamic_local change the origin accordingly
+        //MAC is added/updated as dynamic to allow aging.
+        storeFdbData.origin = FDB_ORIGIN_LEARN;
+        storeFdbData.type = "dynamic";
+    }
 
     m_entries[entry] = storeFdbData;
 
     string key = "Vlan" + to_string(vlan.m_vlan_info.vlan_id) + ":" + entry.mac.to_string();
 
-    if (fdbData.origin != FDB_ORIGIN_VXLAN_ADVERTIZED)
+    if ((fdbData.origin != FDB_ORIGIN_MCLAG_ADVERTIZED) ||
```

Comment:
i think should be &&， right?
 if ((fdbData.origin != FDB_ORIGIN_MCLAG_ADVERTIZED) &&
     (fdbData.origin != FDB_ORIGIN_VXLAN_ADVERTIZED))

Reply:
@pettershao-ragilenetworks : Thanks for pointing to the error. You're right and is corrected now.